src/css/cart.css

```diff
@@ -0,0 +1,14 @@
+@import "../common.css";
```

Comment:
just  a suggestion, dont add files which are unrelated to the PR.

Reply:
this files was needed to render navigation.